src/sql/base/browser/ui/selectBox/selectBox.ts

```diff
@@ -65,6 +67,10 @@ export class SelectBox extends vsSelectBox {
 	private element?: HTMLElement;
 
 	constructor(options: SelectOptionItemSQL[] | string[], selectedOption: string, contextViewProvider: IContextViewProvider, container?: HTMLElement, selectBoxOptions?: ISelectBoxOptions) {
+		if (options.length === 0) {
```

Comment:
this is a behavior change, i think sometimes we do initialize the selectbox with no options and then add options to it. I could be wrong, but if you are making this change, should definitely do some testing.

Reply:
Not an issue anymore